src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
-body {
+$product-info-color: #616070;
+$main-color: #060b35;
+$btn-color: #00acdc;
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-inline: 19px;
```

Comment:
The `margin-inline` property is not standard for the task checklist. Consider using `margin-left` and `margin-right` if you need to specify the horizontal margins explicitly.